doc/modules/changes/20240613_danieldouglas92

```diff
@@ -0,0 +1,4 @@
+Added: Functionality to visualize the fluid
+velocity when not using Melt transport.
```

Comment:
This is not quite specific enough:
```suggestion
velocity when advecting the porosity with the Darcy velocity.
```

Needs another update if you agree with my comment further down.